draft-ietf-tls-tls13.md

```diff
@@ -2183,8 +2183,10 @@ hidden by TCP.
 
 A small allowance for errors in clocks and variations in measurements
 is advisable.  However, any allowance also increases the opportunity
-for replay.  In this case, it is better to reject early data than to
-risk greater exposure to replay attacks.
+for replay.  In this case, it is better to reject early data and fall back
+to a full 1-RTT handshake than to risk greater exposure to replay attacks,
+so in common network topologies, a small allowance on the order of ten
+seconds is reasonable.
```

Comment:
This seems very high. The difference here is causes by errors in RTT estimation and clock speed differentials, which should not be ten seconds over a week.

@siyengar do you have any estimates of how fast client clocks change?